sys/riscv/riscv/pmap.c

```diff
@@ -3689,11 +3686,8 @@ pmap_zero_page(vm_page_t m)
 void
 pmap_zero_page_area(vm_page_t m, int off, int size)
 {
-	vm_pointer_t va = PHYS_TO_DMAP(VM_PAGE_TO_PHYS(m));
+	vm_pointer_t va = PHYS_TO_DMAP_PAGE(VM_PAGE_TO_PHYS(m));
```

Comment:
On Morello you constrained this to the exact bounds of `size`

Reply:
Made this consistent.